core/bazooka.go

```diff
@@ -404,7 +409,7 @@ func (b *Bazooka) SubmitBatch(commitments []Commitment) error {
 		// generate auth
 		auth, err := b.generateAuthObj(b.EthClient, callMsg)
 		if err != nil {
-			b.log.Error("Error creating auth object", "error", err)
+			b.log.Error("Estimate gas failed, tx reverting", "error", err)
```

Comment:
Is this accidentally commited?

Reply:
No this was purposeful, auth message wasnt very clear.